core/services/functions/listener.go

```diff
@@ -499,7 +558,6 @@ func (l *FunctionsListener) handleRequest(ctx context.Context, requestID Request
 	}
 	if userErr != nil {
 		l.logger.Debugw("user error during getSecrets", "requestID", requestIDStr, "err", userErr)
-		fmt.Println("userError", userErr.Error())
```

Comment:
Removed unnecessary log

Reply:
IDK, I saw it when I was removing Print statements I had added during debugging.